config/dynamodb.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
All of this should be in `config/easyaws.php`. Although I guess none of this should be here, because it seems like the whole purpose of this repository is to cache the AWS credentials while leaving the configuration of the client up to the consumer